### PR TITLE
dataset_structure

### PR DESCRIPTION
**Pull request by Rostislav3**
Add description and task 2

### CODE REVIEW

File: MalininR.P/lib.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+def host_name_validator(text):
+
+	if re.search(f'\s',text):
+
+		return False, 'Not Correct input text.Spaces in it'
+
+	else:
+
+		return (True, ) if re.match(f'\w[A-Za-z0-9_]*',text) else (False, 'Not Correct input text')
```
Comment (GasperPaul) [inline MalininR.P/lib.py]: All the space in this file

File: MalininR.P/task2.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
```
Comment (GasperPaul) [inline MalininR.P/task2.py]: This file is not requested by the task.

File: MalininR.P/task4.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        "host_response_time" : 10,
+    },
+
+    "New Modern House-Amazing water view" :{
+        "host_name" : "Jill",
+        "street" : "West Lee Street, Seattle, WA 98119, United States",
+        "host_response_time" : 10
+    }
+
+}
```
Comment (GasperPaul) [inline MalininR.P/task4.py]: Should not be here.

File: MalininR.P/task4.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+dataset['Workstate apartment'] = {
+					'host_name': 'Rostik',
+					'street' : 'Wall street',
+					'host_response_time' : 10,
+
+				}
```
Comment (GasperPaul) [inline MalininR.P/task4.py]: Expected a function.

File: MalininR.P/task4.py [hunk 5]
```diff
@@ -0,0 +1,28 @@
+dataset = {
```
Comment (GasperPaul) [inline MalininR.P/task4.py]: This file name is invalid.

File: MalininR.P/task3.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
+dict_keys = list(dataset.keys())
```
Comment (GasperPaul) [inline MalininR.P/task3.py]: This file name is invalid.

**Comment by igortereshchenko** [comment]
Назва основної теки некоректна. Штраф

**Pull request merged**